src/ajax/check_hgvs_dialogue.php

```diff
@@ -128,15 +124,15 @@
     // We will fill in this fixed variant, close the dialogue,
     //  and perform a new call to this script by activating
     //  the onChange.
-    var oInput = $(\'input[name="' . $sFieldName . '"]\');
-    oInput.val("' . addslashes(lovd_fixHGVS($sVariant)) . '");
+    var oInput = $(\'input[name="' . htmlspecialchars($sFieldName) . '"]\');
```

Comment:
Just like with `$sVariant` earlier, it's probably best to leave this as its `$_REQUEST` counterpart or clean it beforehand. This way, `$sFieldName` is not mistaken for a cleaned variable.